tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md

```diff
@@ -0,0 +1,178 @@
+---
+id: install-openstack-with-conjure-up
+summary: Learn how to deploy OpenStack on a single machine using the conjure-up spell.
+categories: containers
+tags: containers, server, openstack, conjure-up, gci
+difficulty: 2
+status: published
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+published: 2018-01-13
+author: Konrad Krawiec <konrad11901@gmail.com>
+
+---
+
+# Install single-server OpenStack with conjure-up
+
+## Overview
+Duration: 1:00
+
+[OpenStack](https://www.openstack.org) is a very popular software for creating private and public clouds.
+
+OpenStack is quite an advanced platform and deploying it may not be an easy task. Luckily, with [conjure-up](https://conjure-up.io), it's very simple and straightforward.
+
+### What you'll learn
+
+- How to install OpenStack on your machine with conjure-up
+- How to install and configure LXD
+
+### What you'll need
+
+- A well-specced computer running Ubuntu 16.04 LTS with 16 gigabytes of RAM (SSD drive is strongly recommended)
+- Some basic command-line knowledge
+- An hour of your time
```

Comment:
I really like 'An hour of your time' :+1:

Reply:
To be honest, in my case it was like two hours or so 😆